queries/thrift/highlights.scm

```diff
@@ -67,17 +67,15 @@
   (#lua-match? @constant "^[_A-Z][A-Z0-9_]*$"))
 
 (enum_definition
-  "enum"
-  .
-  (identifier) @type
-  "{"
-  (identifier) @constant
-  "}")
+  type: (identifier) @type)
 
-; Builtin Types
-(primitive) @type.builtin
+(enum_definition
+  type: (identifier)
+  (identifier) @constant)
```

Comment:
think this can match identifiers in the enum name, e.g. `bar` in `foo.bar` as well as inside the enum

Reply:
Oops, thank you. Changed it to match after `"{"`